### PR TITLE
Updated the humans_or_horses livebook

### PR DESCRIPTION
**Pull request by meanderingstream**
Added a really simple Exploratory Analysis section and
configuration module.

All hanging functions were put inside modules.  The livebook
can run to completion on my CPU.

### CODE REVIEW

File: notebooks/humans_or_horses.livemd [hunk 1]
```diff
@@ -105,38 +175,42 @@ Axon.dropout and Axon.spatial_dropout create dropout layers which prevent the ne
...
 ```elixir
-defp train_model(model, data, optimizer, epochs) do
-  model
-  |> Axon.Loop.trainer(:binary_cross_entropy, optimizer, log: 1)
-  |> Axon.Loop.metric(:accuracy)
-  |> Axon.Loop.run(data, %{}, epochs: epochs, iterations: 100)
+defmodule HorsesHumans.Training do
+  def train_model(model, data, optimizer, epochs) do
+    model
+    |> Axon.Loop.trainer(:binary_cross_entropy, optimizer, log: 1)
```
Comment (seanmor5) [inline notebooks/humans_or_horses.livemd:210]: Axon's BCE is specifically designed for the case of an output with a single probability between 0 and 1---I would replace this with `:categorical_cross_entropy`.

**Review by seanmor5 — COMMENTED** [review]
Looks good! Just some minor comments :)

File: notebooks/humans_or_horses.livemd [hunk 2]
```diff
@@ -8,61 +8,131 @@ In this notebook, we want to predict whether an image presents a human or a hors
...
+
+<!-- livebook:{"break_markdown":true} -->
+
+Let's look at a sample image.
 
-  {:ok, binary, shape, :u8, :rgba} = StbImage.from_file(filename)
+```elixir
+Enum.at(sample_data, 0)
+|> StbImage.read_file!()
+|> StbImage.to_binary(:png)
```
[the remark this answers is not in the document]
Comment (meanderingstream) [reply notebooks/humans_or_horses.livemd:77]: No, it just writes the binary.  I wasn't sure what to use to see the image.  I'm having trouble seeing Vegalite working on my local LiveBook server.  I was hesitant to add another dependency.

**Comment by meanderingstream** [comment]
I believe that I've made the changes that were requested.

**Pull request merged**